### DISCUSSION TITLE
Is there a way to show full citation in the body of text instead of in a bibliography section?

### DISCUSSION THREAD

**Discussion by victorfeagins** [quarto-dev/quarto-cli#4414] [Q&A] [answered]
I was reading this https://quarto.org/docs/authoring/footnotes-and-citations.html

I was curious if I could instead of citation showing as it would as a in text citation, but it would show the entire citation as it would appear in the bibliography. 

I am creating a type of annotated bibliography document and would like to have the full citation then the content below. Might it be possible to use the div but make it only show one citation?
::: {#refs}
:::

**Comment by mcanouil** [comment]
Using only base Quarto you can't.
But you can use a Lua filter which could be highly inspired by <https://github.com/pandoc-ext/section-bibliographies> or using <https://github.com/pandoc-ext/multibib>.

**Comment by victorfeagins** [comment, marked as answer]
Ah those are two handy extensions. The multibib could work but it would require me to make x number of bib files for each annotation bibliography. 

I looked around and it seems for my purposes creating a custom csl file as seen here that makes in text citations look like full text citations.

As seen in this blog: <https://www.andrewheiss.com/blog/2023/01/09/syllabus-csl-pandoc/#:~:text=The%20easiest%20way%20to%20get%20full%20citations%20inline,files%20contains%20a%20%3Cstyle%3EXML%20element%20with%20a%20classattribute.>
